yocto-dev/scripts/gtsam_som_test/test_gtsam.cpp

```diff
@@ -0,0 +1,107 @@
+/* ----------------------------------------------------------------------------
```

Comment:
Do you think this is useful to include this directory in this PR?

Reply:
Not really, will remove and add later.